src/validation.cpp

```diff
@@ -773,15 +773,17 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
         constexpr unsigned int scriptVerifyFlags = STANDARD_SCRIPT_VERIFY_FLAGS;
 
         // Check against previous transactions
-        // This is done last to help prevent CPU exhaustion denial-of-service attacks.
+        // The first loop above does all the inexpensive checks.
```

Comment:
This is a really confusing comment here. "The first loop" refers to the checks inside `CheckTxInputs()` call on L566, which has been moved around since the comment was first written.

I'll fix this comment in a follow-up PR.

Reply:
Being addressed in #16658.